js/pages/incidence-rates/tabs/export/bigquery.js

```diff
@@ -0,0 +1,20 @@
+define([
+	'text!./bigquery.html',
+	'components/Component',
+	'utils/AutoBind',
+	'utils/CommonUtils',
+], function (
+	view,
+	Component,
+	AutoBind,
+	commonUtils,
+) {
+	class IRexportbigqueryTab extends AutoBind(Component) {
```

Comment:
Why do you need separate components for each dialect? What's difference between e.g. `IRexportbigqueryTab` and `IRexportimpalaTab`? Why cannot a single component be used?

Reply:
done